tsl/src/compression/create.c

```diff
@@ -923,24 +923,54 @@ disable_compression(Hypertable *ht, WithClauseResult *with_clause_options)
 }
 
 /* Add column to internal compression table */
-static void
-add_column_to_compression_table(Hypertable *compress_ht, CompressColInfo *compress_cols)
+static bool
+add_column_to_compression_table(Hypertable *compress_ht, CompressColInfo *compress_cols,
+								bool missing_ok)
 {
 	Oid compress_relid = compress_ht->main_table_relid;
 	ColumnDef *coldef;
 	AlterTableCmd *addcol_cmd;
+	HeapTuple attTuple;
+	int attnum;
+	bool skipped = false; /* postgres operates on its table first, so the column may have been
```

Comment:
You can't really assume this here. Maybe this function is going to be called from a different place in the future. There is no guarantee that postgres operated on its table first.